extension/data/modules/profile.js

```diff
@@ -851,6 +876,47 @@ function profilepro () {
             }
             makeProfile(user, listing, options);
         });
+
+        // Open profile bashed on hash params
+        window.addEventListener('TBHashParams', event => {
+            const listing = event.detail.tbprofile;
+            if (listing && listingTypes.includes(listing)) {
+                let user;
+                // If we get a user from the params we go from there.
+                if (event.detail.user) {
+                    user = event.detail.user;
+
+                // If not we check if we are on a profile page.
+                // We can safely check this object as tbNewPage logic is done before TBHashParams logic.
+                } else if (TBCore.pageDetails.pageType === 'userProfile') {
+                    user = TBCore.pageDetails.pageDetails.user;
+
+                // Finally we simply return if we have no username to work with.
+                } else {
+                    TB.ui.textFeedback('No user present in parameters and not on profile page.', TBui.FEEDBACK_NEGATIVE);
+                    return;
+                }
+
+                const options = {
+                    sort: event.detail.sort || 'new',
```

Comment:
`sort` is defaulted to `'new'` in the definition of `makeProfile`, doesn't need to be defaulted here too
```suggestion
                    sort: event.detail.sort,
```